.github/actions/templates/avm-validateModuleDeployment/action.yml

```diff
@@ -62,8 +62,11 @@ runs:
     - name: Azure Login
       uses: azure/login@v2
       with:
+        client-id: ${{ env.AZURE_CLIENT_ID }}
```

Comment:
Unrelated changes. Please undo

Reply:
Done. Please consider reopening/merging https://github.com/Azure/bicep-registry-modules/pull/2523. It contains just the changes mentioned here and does not at all conflict with current workflows.